src/test/java/org/folio/linked/data/preprocessing/lccn/SpaceAdderStructureaTest.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.linked.data.preprocessing.lccn;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.regex.Pattern;
+import org.folio.spring.testing.type.UnitTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+@UnitTest
+class SpaceAdderStructureaTest {
```

Comment:
Structurea isn't that readable. Let's name it like StructureA here and in StructureB test too?

Reply:
I have named these classes like so because checkstyle doesn't allow to call them like SpaceAdderStructure**A**Test/SpaceAdderStructure**B**Test

Reply:
Checkstyle makes it worse, that's a pity.